src/search/FAQSearchIndex.php

```diff
@@ -1,4 +1,14 @@
 <?php
```

Comment:
The `FAQPage_Controller` file above has a doc block on the class that is empty.

Also it was throwing errors on `dev/build` as it could not find a `PageController` class.

Reply:
done